README.md

```diff
@@ -11,13 +11,16 @@ You need to have [virtualenv](https://virtualenv.pypa.io/en/stable/) installed.
 
 Then simply run `make serve`.
 
-## Deploying
+## Contributing
 
-Raise a pull request. Once the PR is approved and merged, [readthedocs][project] will automatically rebuild the docs.
```

Comment:
why not keep this?

Reply:
I had moved most of this text at the end of "Making A Change" section in the new CONTRIBUTE.md. Thought it best to move, as otherwise someone may just raise a PR without reading the contents in the contrib page?